### PR TITLE
Support for pip

### PR DESCRIPTION
**Pull request by prusse-martin**

### CODE REVIEW

**Comment by prusse-martin** [comment]
Adding python to the `requirements_dev.txt` cause the [build to fail](https://travis-ci.org/ESSS/conda-devenv/builds/277315115).

File: conda_devenv/devenv.py [hunk 1]
```diff
@@ -78,47 +100,61 @@ def merge(dicts, keys_to_skip=('name',)):
...
+    new_dict_dependencies = []
+    for dep in value_to_merge:
+        if isinstance(dep, dict):
+            for key in dep:
+                merge_dependencies_version_specifications(dep, key_to_merge=key)
+            new_dict_dependencies.append(dep)
+        elif isinstance(dep, six.string_types):
+            m = re.match(package_pattern, dep)
+            if m is None:
+                raise RuntimeError('The package version specification "{}" do not follow the'
```
Comment (nicoddemus) [inline conda_devenv/devenv.py]: To make sure: do we have a test for this condition?

File: conda_devenv/devenv.py [hunk 2]
```diff
@@ -78,47 +100,61 @@ def merge(dicts, keys_to_skip=('name',)):
...
-                              "Only lists and dicts can be merged. The type obtained was: {type}"\
-                        .format(
-                            key=key,
-                            type=type(value)
-                        )
+                    message = ' '.join([
+                        "Can't merge the key: '{key}' because it will override the previous value.",
+                        "Only lists and dicts can be merged. The type obtained was: {type}",
+                    ]).format(key=key, type=type(value))
                     raise ValueError(message)
```
Comment (nicoddemus) [inline conda_devenv/devenv.py]: To make sure: do we have a test for this condition?

**Comment by prusse-martin** [comment]
Merging and tagging as 1.0.0

**Pull request merged**